tests/FSharp.Compiler.ComponentTests/Conformance/DelegateTypes/invalid_delegate_definition.fs.err.bsl

```diff
@@ -0,0 +1 @@
+﻿invalid_delegate_definition.fs (1,6)-(1,15) No argument are allowed in delegate primary constructor.
```

Comment:
@vzarytovskii Is it expected that the baseline doesn't contain the error numbers? Or is it checked in some other place?

I think it could be a good thing to check that numbers aren't changing during some changes in analysis code.

Reply:
I think in these baselines it is, I will need to fix it.